packages/perseus/src/widgets/phet-simulation/phet-simulation.tsx

```diff
@@ -240,9 +240,9 @@ const styles = StyleSheet.create({
         borderRadius: borderRadiusLarge,
         borderWidth: 1,
         borderColor: basicBorderColor,
-        padding: phoneMargin,
+        padding: spacing.medium_16,
         paddingBottom: 0,
-        width: 650,
+        width: 400 + spacing.medium_16 * 2,
```

Comment:
Would `spacing.xLarge_32` make sense here, rather than multiplying?

Reply:
Brilliant, and I will include a comment explaining why the xLarge_32!